R/post-observer.R

```diff
@@ -105,10 +125,10 @@ make_gbm_predictions = function(x, use_obs_model){
     mean = mean + test$observer_effect
   }
   
-  cbind(test, mean = mean, model = "richness_gbm",  use_obs_model = use_obs_model, 
+  cbind(test, mean = mean, model = "richness_gbm", 
         stringsAsFactors = FALSE) %>% 
-    select(site_id, year, mean, richness, model, use_obs_model) %>% 
-    mutate(sd = sd)
+    select(site_id, year, mean, richness, model) %>% 
+    mutate(sd = !!sd, n.trees = !!n.trees, use_obs_model = !!use_obs_model)
```

Comment:
Thanks for the earlier explanation of `!!`. I understood (I think) the uses on lines 89 and 92, but I'm confused here. Where are the external `sd` and `n.trees` coming in. I don't see them passed to the function or in the above lines. If they are in `test` then I don't understand the use of `!!`.